### ISSUE TITLE
Emails: Research how track responses

### ISSUE THREAD

**Issue by leandrosardi** [leandrosardi/emails#13]
## Plan

1. :heavy_check_mark: make `reply_to` not mandatory.
2. :x: Design data model.
3. :x: Map responses with CS Google Integration.
4. :x: Map responses with GMass Bridge.

## Plan for Mapping Responses with CS Google Integration
1. :x: Add a `message_id` to each delivery, in order to track responses later.
2. :x: Send a campaign to `leandro.sardi@expandedventure.com`, with no `reply-to`, using CS Google Integration.
3. :x: Write a process to process incoming emails from both: the inbox and spam folder, and link them to the `delivery` thru the `message_id`.

**Comment by leandrosardi** [comment]
## Building List of 1 Lead

```sql
select id_lead from fl_data where value like 'leandro.sardi@expandedventure.com'
-- c8e2a984-2104-4902-863e-7ed9255f074d

select * from fl_export where "filename"='testing imap'
-- e5710196-f805-4ea5-9395-4683e36ae5a8

select * from fl_export_lead where id_export='e5710196-f805-4ea5-9395-4683e36ae5a8'
--update fl_export_lead set id_lead='c8e2a984-2104-4902-863e-7ed9255f074d' where id_export='e5710196-f805-4ea5-9395-4683e36ae5a8'
```

**Comment by leandrosardi** [comment]
I resolved it on leandrosardi/emails#1
